### PR TITLE
Add a simple client test

### PR DESCRIPTION
**Pull request by tshak**
This makes life easier for running a quick integration test to ensure that BaaS is setup correctly.

### CODE REVIEW

File: client-test.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+var BaasClient = require('./pool');
+
+BAAS_HOST=process.env.BAAS_HOST || 'baas.svc.dev-us-east-1.platform.auth0.com'
```
Comment (elbuo8) [inline client-test.js]: Don't think we should have this default.

Comment (elbuo8) [reply client-test.js]: This is a public repo after all.

Comment (tshak) [reply client-test.js]: Oh... right

File: client-test.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+var BaasClient = require('./pool');
+
+BAAS_HOST=process.env.BAAS_HOST || localhost;
```
Comment (elbuo8) [inline client-test.js]: `localhost`

**Pull request merged**